src/components/LoginDropdown.js

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import { Menu, Transition } from "@headlessui/react";
+import { personCircleSharp } from "ionicons/icons";
+import { IonIcon } from "@ionic/react";
+import Button from "./Button";
+
+export default function Dropdown({ onClick = () => {} }) {
+  return (
+    <div className="flex items-center justify-center">
+      <div className="relative inline-block text-left">
+        <Menu>
+          {({ open }) => (
+            <>
+              <span className="rounded-md shadow-sm">
+                <Menu.Button className="inline-flex justify-center w-full text-sm font-medium leading-5 rounded-md">
+                  <Button onClick={onClick}>
+                    <IonIcon icon={personCircleSharp} size="small"></IonIcon>
+                  </Button>
```

Comment:
Since clicking on the user icon won't do anything, because `{ onClick = () => {} }` if `src/components/LoginDropdown.js` doesn't receive an `onClick` prop, and it doesn't, should we remove the button element entirely, and just leave `<IonIcon icon={personCircleSharp} size="small"></IonIcon>` in its place?